Gem/Code/Source/DiffuseGIExampleComponent.cpp

```diff
@@ -384,8 +384,8 @@ namespace AtomSampleViewer
         
         m_directionalLightPitch = AZ::DegToRad(-65.0f);
         m_directionalLightYaw = AZ::DegToRad(65.0f);
-        m_directionalLightColor = AZ::Color(0.92f, 0.78f, 0.35f, 1.0f);
-        m_directionalLightIntensity = 30.0f;
+        m_directionalLightColor = AZ::Color(1.0f, 0.9f, 0.6f, 1.0f);
+        m_directionalLightIntensity = 20.0f;
```

Comment:
Do you need to update any screenshots with this change?

Reply:
The Sponza scene is not currently part of the test.